pkg/ipmi/ipmi.go

```diff
@@ -32,12 +33,24 @@ const (
 	_IPMI_OPENIPMI_READ_TIMEOUT      = 15
 	_IPMI_SYSTEM_INTERFACE_ADDR_TYPE = 0x0c
 
-	_BMC_SET_WATCHDOG_TIMER = 0x24
-	_BMC_GET_WATCHDOG_TIMER = 0x25
-
 	_IPM_WATCHDOG_NO_ACTION    = 0x00
 	_IPM_WATCHDOG_SMS_OS       = 0x04
 	_IPM_WATCHDOG_CLEAR_SMS_OS = 0x10
+
+	_SYSTEM_INFO_BLK_SZ = 16
+
+	_SYSTEM_FW_VERSION = 1
+
+	_ASCII = 0
+
+	// Set 62 Bytes (4 sets) as the maximal string length
+	strlenMax = 62
+)
+
+const (
```

Comment:
Why using a another `const` section?

Reply:
I misunderstood your previous comment
> How about grouping all the commands (_BMC_SET_WATCHDOG_TIMER, _BMC_GET_WATCHDOG_TIMER and _SET_SYSTEM_INFO_PARAMETERS) together?
So you meant put these three commands in a group within this const section, right?

Reply:
I meant to say to keep/group them together, in the same block.

Reply:
Updated.